spec/rubocop/cop/rails/set_ignored_columns_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::SetIgnoredColumns, :config do
+  it 'registers an offense when using `ignored_columns` once' do
+    expect_offense(<<~RUBY)
+      class User < ActiveRecord::Base
+        self.ignored_columns = [:one]
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ This assignment to `ignored_columns` may overwrite previous ones.
```

Comment:
Can you tweak the offense range and message?

```suggestion
        self.ignored_columns = [:one]
                             ^ Use `+=` instead of `=`.
```